.github/CODEOWNERS

```diff
@@ -0,0 +1,76 @@
+/src/compute                        @MaterializeInc/compute
+/src/compute-client                 @MaterializeInc/compute
+/src/environmentd                   @MaterializeInc/surfaces
+/src/expr                           @MaterializeInc/compute
+/src/expr-test-util                 @MaterializeInc/compute
+/src/frontegg-auth                  @MaterializeInc/compute
+/src/http-util                      @MaterializeInc/surfaces
+/src/interchange                    @MaterializeInc/storage
+/src/kafka-util                     @MaterializeInc/storage
+/src/kinesis-util                   @MaterializeInc/storage
+/src/lowertest                      @MaterializeInc/compute
+/src/lowertest-derive               @MaterializeInc/compute
+/src/metabase                       @benesch
+/src/mz                             @MaterializeInc/surfaces
+/src/npm                            @benesch
+/src/orchestrator                   @benesch
+/src/orchestrator-kubernetes        @benesch
+/src/orchestrator-process           @benesch
+/src/orchestrator-tracing           @benesch
+/src/ore                            @benesch
+/src/persist                        @MaterializeInc/persist
+/src/persist-client                 @MaterializeInc/persist
+/src/persist-types                  @MaterializeInc/persist
+/src/pgcopy                         @MaterializeInc/surfaces
+/src/pgrepr                         @MaterializeInc/surfaces
+/src/pgtest                         @MaterializeInc/surfaces
+/src/pgwire                         @MaterializeInc/surfaces
+/src/pid-file                       @benesch
+/src/postgres-util                  @MaterializeInc/surfaces
+/src/prof                           @umanwizard
+/src/proto                          @aalexandrov
+/src/repr                           @MaterializeInc/storage @MaterializeInc/compute
+/src/repr-test-util                 @MaterializeInc/compute
+/src/s3-datagen                     @MaterializeInc/storage
+/src/secrets                        @MaterializeInc/cloud
+/src/segment                        @benesch
+/src/service                        @MaterializeInc/storage @MaterializeInc/compute
+/src/sql                            @MaterializeInc/surfaces
+/src/sql-parser                     @MaterializeInc/surfaces
+/src/sqllogictest                   @MaterializeInc/surfaces
+/src/ssh-util                       @MaterializeInc/storage
+/src/stash                          @MaterializeInc/surfaces
+/src/stash-debug                    @MaterializeInc/surfaces
+/src/storage                        @MaterializeInc/storage
+/src/storage-client                 @MaterializeInc/storage
+/src/testdrive                      @MaterializeInc/qa
+/src/timely-util                    @MaterializeInc/compute
+/src/transform                      @MaterializeInc/compute
+/src/walkabout                      @benesch
+/src/workspace-hack                 @benesch
+/test
```

Comment:
One thing I had trouble deciding was whether qa should own `test/`, or if notifications from test changes would be too spammy.

Reply:
cc @philip-stoev

Reply:
No problem lets keep it this way.